enatega-multivendor-admin/src/components/Rider/Rider.jsx

```diff
@@ -223,6 +230,20 @@ function Rider(props) {
                     : ''
                 ]}
               />
+              {usernameError === false && (
```

Comment:
usernameError will be an object return from the validate function. So, this statement will not get rendered even if the error exists.
Initially it will be null and then it will have object of {isValid, errorMessage}. Please tackle null and object case. usernameError === false does not make sense here.

Reply:
The usernameError variable is a simple state used for onSubmit validation. Its only purpose is to indicate whether an error exists (true) or not (false). Throughout the code, you'll notice that most error handling conditions involve checking whether that particular state is true or false. I have set the only isValid property to the usernameError not message so it won't be a object.

Reply:
You're welcome, and I'm excited to contribute more in the future.